lib/tier-utils.js

```diff
@@ -104,7 +107,8 @@ export const sortTiers = (baseTiers, orderKeys, hasCustomContribution) => {
 export const sortTiersForCollective = (collective, baseTiers) => {
   const orderKeys = getCollectiveContributionCardsOrder(collective);
   const hasCustomContribution = !collective.settings?.disableCustomContributions;
```

Comment:
Consistency, rewrite `hasCustomContribution` to have the same syntax than `hasCryptoContribution`. They're next to each others and it's suprising to not have the same code style.